src/test/kotlin/com/cognifide/gradle/aem/common/utils/FileUtilTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.cognifide.gradle.aem.common.utils
+
+import com.cognifide.gradle.aem.common.utils.FileUtil.sanitizePath
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class FileUtilTest {
+
+    @Test
+    fun `should sanitize path properly`() {
```

Comment:
How about a test that will say that path is wrong.

Reply:
Sure but no case in real time 😉 gap is not a framework 😁